src/openrct2/world/Park.cpp

```diff
@@ -773,12 +773,14 @@ void Park::UpdateHistories()
     {
         currentWeeklyProfit /= gameState.WeeklyProfitAverageDivisor;
     }
-    HistoryPushRecord<money64, FINANCE_GRAPH_SIZE>(gameState.WeeklyProfitHistory, currentWeeklyProfit);
+    HistoryPushRecord<money64, std::extent_v<decltype(GameState_t::WeeklyProfitHistory)>>(
+        gameState.WeeklyProfitHistory, currentWeeklyProfit);
```

Comment:
Legibility might be improved by instead doing something like:
```suggestion
    constexpr auto profitHistorySize = std::extent_v<decltype(GameState_t::WeeklyProfitHistory)>;
    HistoryPushRecord<money64, profitHistorySize>(gameState.WeeklyProfitHistory, currentWeeklyProfit);
```